packages/shared-models/src/api.ts

```diff
@@ -71,28 +80,32 @@ export interface ISharedText extends ISharedBase {
   /**
    * The changed signal.
    */
-  readonly changed: ISignal<this, TextChange>;
+  readonly changed: ISignal<this, SourceChange>;
+
   /**
-   * Gets cell's source.
+   * Text
+   */
+  source: string;
```

Comment:
Using properties instead of getter and setter functions is a more common pattern in JLab.
I did not remote the getter/setter though to ease backport to 3.x.